client/src/components/game/Game.tsx

```diff
@@ -53,141 +55,168 @@ const initialState: IGameState = {
-                  }`}
-                key={i}
-              >
-                <div className="player-info">
-                  <img
-                    className="player-avatar"
-                    src={`${config.githubAvatarUri}${player.username}`}
-                  />
-                  <div>{player.username}</div>
-                </div>
-
-                <div>{player.score}</div>
-              </div>
-            ))}
-          </div>
-          {state.state === SessionState.IN_LOBBY && <Editor chunk={lobbyChunk} />}
-          {state.state === SessionState.IN_GUESSING && (
-            <> 
-              <Timer expiration={(state.prompt as IPrompt).guessExpiration}/>
-              <Editor chunk={(state.prompt as IPrompt).chunk} />
+        <div className="right-panel">
+          {inGuessing() && (
+            <>
+              <Timer expiration={(state.prompt as IPrompt).guessExpiration} />
             </>
           )}
-          {state.state === SessionState.DONE_GUESSING && (
-            <Answer
-              correctChoice={(state.answer as IAnswer).correctChoice}
-              players={(state.answer as IAnswer).players}
-              outOfChunks={false}
-            />
-          )}
-          {state.state === SessionState.OUT_OF_CHUNKS && (
-            <Answer
-              correctChoice={(state.answer as IAnswer).correctChoice}
-              players={SessionService.getSortedPlayers(state.players)}
-              outOfChunks={true}
-            />
-          )}
+          <button onClick={copyHandler} className="game-buttons">
+            Copy
+          </button>
         </div>
-        {state.state === SessionState.IN_GUESSING && (
-          <div className="choices">
-            {(state.prompt as IPrompt).choices.map((choice: string, i: number) => (
-              <button className="choice" onClick={(event) => guessHandler(event)}>
-                {choice}
-              </button>
-            ))}
-          </div>
-        )}
+      </div>
+      <div className="mid">
+        <div className="players-container">
+          {state.players.map((player: IPlayer, i: number) => (
+            <div
+              className={`player ${isHost(player.username) ? "host" : ""}`}
+              key={i}
+            >
+              <div className="player-info">
+                <img
+                  className="player-avatar"
+                  src={`${config.githubAvatarUri}${player.username}`}
+                />
+                <div>{player.username}</div>
+              </div>
 
-        <Notification />
-      </>
-    );
+              <div>{player.score}</div>
+            </div>
+          ))}
+        </div>
+
+        {inLobby() && <Editor chunk={lobbyChunk} />}
+        {inGuessing() && (
+          <>
+            <Editor chunk={(state.prompt as IPrompt).chunk} />
+          </>
+        )}
+        {inDoneGuessing() && (
+          <Answer
```

Comment:
Are you going to show the repository after they guessed? Will this be in a separate commit?

Reply:
Separate pr